tests/python/conftest.py

```diff
@@ -14,21 +14,22 @@
 
 
 @pytest.fixture(autouse=True, scope="function")
-def amrex_init():
-    amr.initialize(
-        [
-            # print AMReX status messages
-            "amrex.verbose=2",
-            # throw exceptions and create core dumps instead of
-            # AMReX backtrace files: allows to attach to
-            # debuggers
-            "amrex.throw_exception=1",
-            "amrex.signal_handling=0",
-            # abort GPU runs if out-of-memory instead of swapping to host RAM
-            "amrex.abort_on_out_of_gpu_memory=1",
-            # do not rely on implicit host-device memory transfers
-            "amrex.the_arena_is_managed=0",
-        ]
-    )
+def amrex_init(tmpdir):
+    with tmpdir.as_cwd():
+        amr.initialize(
+            [
+                # print AMReX status messages
+                "amrex.verbose=2",
+                # throw exceptions and create core dumps instead of
+                # AMReX backtrace files: allows to attach to
+                # debuggers
+                "amrex.throw_exception=1",
+                "amrex.signal_handling=0",
+                # abort GPU runs if out-of-memory instead of swapping to host RAM
+                "amrex.abort_on_out_of_gpu_memory=1",
+                # do not rely on implicit host-device memory transfers
+                "amrex.the_arena_is_managed=0",
+            ]
+        )
     yield
     amr.finalize()
```

Comment:
According to @sayerhs in https://github.com/AMReX-Codes/pyamrex/pull/181#discussion_r1308006503
```suggestion
        yield
        amr.finalize()
```